client_app/src/components/shelter/EditStatusConfirmationModal.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'; 
+import { ModalComponent } from './ModalComponent';
+
+export const EditStatusConfirmationModal = props => {
+  const {isStatusModalOpen, setIsStatusModalOpen, shift} = props;
+
+  const onYesClick = () => {
+    console.log("changed shift status to: " + !shift.status)
```

Comment:
No, I was creating the methods so that minimal stuff needs to be changed once we integrate with the backend, that's why I have all the methods with the print statements.

Reply:
So I was leaving all the console log statements, they aren't really for debugging but representing functionality

Reply:
Thank you for the detailed explanation on why the console.log() statements are in place! It makes sense to have these placeholders until we integrate the backend. I would recommend adding some TODO comments next to the console.log() statements to indicate that they will eventually be replaced by actual backend logic. This will also help other contributors understand their purpose, and as a best practice, we could consider wrapping them in environment checks.

Reply:
Updated that